features.bzl

```diff
@@ -97,6 +97,8 @@ _rules = struct(
     instrumented_files_info_has_metadata_files = ge("7.0.0-pre.20230710.5"),
     # Whether treeartifacts can have symlinks pointing outside of the tree artifact. (#21263)
     permits_treeartifact_uplevel_symlinks = ge("7.1.0"),
+    # Whether rule extension APIs are available by default
+    rule_extension_apis_available = ge("8.0.0"),
```

Comment:
nit: would `8.0.0` include 8.0.0rc1? I forgot @fmeum

Reply:
Yes, as far as I remember we made the check ignore RC suffixes so that we can test more realistically

Reply:
Let's wait for https://github.com/bazel-contrib/bazel_features/pull/81.

Reply:
Turns out we only did this for Bazel compatibility, not bazel_features. Which I guess makes sense.

Reply:
we can just do `ge("8.0.0rc1")`, right?

Reply:
Yes, that's what we should do.

Reply:
Updated to `8.0.0rc1`